core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -133,6 +133,7 @@ type dbReceiptPlus struct {
 	ID           uuid.UUID        `db:"pipeline_task_run_id"`
 	Receipt      evmtypes.Receipt `db:"receipt"`
 	FailOnRevert bool             `db:"FailOnRevert"`
+	State        string           `db:"state"`
```

Comment:
Why do we need this field now if we're always querying only `finalized` txs?

Reply:
oops, forgot to remove it, it's for testing